docroot/modules/custom/uiowa_core/uiowa_core.module

```diff
@@ -457,3 +459,40 @@ function uiowa_core_preprocess_block(&$variables) {
       break;
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ */
+function uiowa_core_node_presave(Node $node) {
+
+  $nodeType = $node->getType();
+
+  switch ($nodeType) {
+    case 'article':
+      $tag_display_config = \Drupal::configFactory()->getEditable('sitenow_articles.settings')->get('tag_display');
+      $related_display_config = \Drupal::configFactory()->getEditable('sitenow_articles.settings')->get('related_display');
+      break;
+
+    case 'page':
+      $tag_display_config = \Drupal::configFactory()->getEditable('sitenow_pages.settings')->get('tag_display');
+      $related_display_config = \Drupal::configFactory()->getEditable('sitenow_pages.settings')->get('related_display');
+      break;
+
+    case 'person':
+      $tag_display_config = \Drupal::configFactory()->getEditable('sitenow_people.settings')->get('tag_display');
+      $related_display_config = \Drupal::configFactory()->getEditable('sitenow_people.settings')->get('related_display');
+      break;
+
+    default:
+      $tag_display_config = 'do_not_display';
+      $related_display_config = 'do_not_display';
+  }
```

Comment:
This could be replaced with:
```php
$node_type = $node->getType();
$type_map = [
  'article' => 'articles', 
  'page' = 'pages', 
  'person' => 'people'
];
$tag_display_config = (isset($type_map[$node_type])) ? \Drupal::configFactory()->getEditable('sitenow_{$node_type}.settings')->get('tag_display') : 'do_not_display';
$related_display_config = (isset($type_map[$node_type])) ? \Drupal::configFactory()->getEditable('sitenow_{$node_type}.settings')->get('related_display') : 'do_not_display';
```

Reply:
I changed `$nodeType` => `$node_type` because I thought it was the standard inside `.module` files, but that is not the case anymore. However it is recommended that we use consistent naming pattern for variables and most use snake case in `.module` files currently: https://www.drupal.org/docs/develop/standards/coding-standards#s-functions-and-variables

Reply:
Working on this. Should update the PR soon after a meeting. Thanks @pyrello

Reply:
@pyrello This has been refactored: https://github.com/uiowa/uiowa/pull/5226/commits/03d014a28e250d99844aff74f144c0c7d741e5c4

I had to adjust the lines setting `$tag_display_config` and `$related_display_config` to use the new`$type_map` and changed the quotes to double quotes so variables are read correctly: 

```php
  $tag_display_config = (isset($type_map[$node_type])) ? \Drupal::configFactory()->getEditable("sitenow_$type_map[$node_type].settings")->get('tag_display') : 'do_not_display';
  $related_display_config = (isset($type_map[$node_type])) ? \Drupal::configFactory()->getEditable("sitenow_$type_map[$node_type].settings")->get('related_display') : 'do_not_display';
```

This seems to work, okay to resolve?

Reply:
Also tested on non-standard content types just to make sure saving doesn't break there.